### PR TITLE
Academy Role

### PR DESCRIPTION
**Pull request by chrisblackburn**

### CODE REVIEW

File: roles/engineer.md [hunk 1]
```diff
@@ -18,6 +16,21 @@ We expect someone in the role:
 - to be autonomously delivering features of a small nature within 3 months
 - to be autonomously delivering features of a medium nature within 6 months
 
+
+## Academy role
```
Comment (craigjbass) [inline roles/engineer.md]: Not sure about the name of this role `Academy`. `Apprentice Engineer` ?

Comment (lukemorton) [inline roles/engineer.md]: `Academy Engineer`

Comment (craigjbass) [inline roles/engineer.md]: What does the qualifier `Academy` mean? Would it require further explanation?

File: roles/engineer.md [hunk 2]
```diff
@@ -18,6 +16,21 @@ We expect someone in the role:
...
+
+## Academy Engineer role
+
+We need someone:
+
+- to train to be ready to join a delivery team following graduation
+
+We expect someone in the role:
+
+- to be autonomously delivering features of a micro nature before graduation
```
Comment (RyanMacG) [inline roles/engineer.md]: Have we got any better language for this? Agree with what you said earlier about it being unclear.

Comment (craigjbass) [inline roles/engineer.md]: "low complexity" => implications on size of domain, customer's understanding of their own domain / needs / desires? Solution for the problem is of low technical complexity - requires very few moving parts i.e. data is already in near-enough correct format, need presentational work?

File: roles/engineer.md [hunk 3]
```diff
@@ -15,8 +13,23 @@ We expect someone in the role:
 - to be fully billable after 2 weeks on-boarding
 - to be involved in practice improvement discussions within 3 months
 - to be actively practicing modern software craftsmanship practices such as TDD within 3 months
-- to be autonomously delivering features of a small nature within 3 months
-- to be autonomously delivering features of a medium nature within 6 months
+- to be autonomously delivering features of modereate complexity within 3 months
```
Comment (craigjbass) [inline roles/engineer.md]: `moderate`

**Pull request merged**